community/index/src/main/java/org/neo4j/index/internal/gbptree/ConsistencyChecker.java

```diff
@@ -17,22 +17,23 @@
  * You should have received a copy of the GNU General Public License
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
-package org.neo4j.index.gbptree;
```

Comment:
Mostly this is to not commit to a public surface since this is purely internal stuff a.t.m.